integration/apps/nekbone/nekbone.py

```diff
@@ -76,7 +76,10 @@ def trial_setup(self, run_id, output_dir):
 
     def experiment_teardown(self, output_dir):
         file_name = os.path.join(output_dir, self._input_name)
-        os.unlink(file_name)
+        try:
+            os.unlink(file_name)
+        except FileNotFoundError:
```

Comment:
I don't understand how it's possible that the data.rea file does not exist when experiment_teardown() is called.  It should be created (or just checked to make sure is still exists) with every trial_setup().  Is there another code path somewhere that removes it?

Reply:
It's probably worth retesting this.  I thought that this function was getting called multiple times, and that's why the file sometimes didn't exist, but experiment_teardown implies that it only happens once per experiment.

Reply:
Ok, I just looked at this and it can get called multiple times.  For example, we might be running an experiment like the barrier_frequency_sweep where there are multiple configs of the same app (which will use the same input file name).